docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+
+To install your project:
+
+  1. Open **Command Palette** and enter **Install Tizen App**.
+  2. Enter the folder name of the project you want to install. (Folder name, not path)
+  3. When the installation is complete, it succeeds, and there is successful generation of the installation app. Also, you will see a build success log in the **terminal** at the bottom on the vscode.
+
+  ![Install your tizen app](media/tizen_web14.png)
+
+> **Note** 
+> 
+> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+
+To run your project:
+
+  1. Open **Command Palette** and enter **Run Tizen App**.
+  2. Enter the folder name of the project you want to run. (Folder name, not path)
+  3. When the run is complete, it succeeds, and there is successful generation of launch app popup. Also, you will see a build success log in the terminal at the bottom on the vscode.
+
+  ![Run Your Tizen App](media/tizen_web15.png)
+
+> **Note**  
+>
+> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+>
+> In addition, vscode terminal displays successful launch log messages:
+> 
+> 1. debug 0 means normal launch.
+> 2. debug 1 means launching an Inspector available.
+
+### Debug Application in Emulator
+
+To debug application, follow these steps:
+
+> **Note**
+>
+> Ensure the following:
+> - Emulator is launched
+> - Device is connected to the PC
+> - Chrome browser is installed
+
+1. Open **Command Palette** and enter **Debug Tizen App**.
+2. Enter the name of the Tizen Web App you want to run.
+3. A successful popup is created and the app is launched.
+
+> **Note**
+> 
+> In addition, `vscode` terminal displays successful launch log messages:
+> vscode Terminal displays a successful launch log
+>
+> - debug `0` means normal launch.
+> - debug `1` means launching an inspector available from command line.
+> - Additionally, debug the connected port as shown in the following log.
+
+  ![Debugging you application](media/tizen_web16.png)
+
+4. Connect **Debug Tizen App** with **Inspector**.
+5. Open **Command Palette** and enter **Run Inspector**.
+6. Enter the port number on your local PC, for example, 9999
+7. In Chrome browser, the Inspector automatically launches and associates with the launched app.
+
+    ![Debugging you application](media/tizen_web17.png)
+
+### Debug Application in Web Simulator
+
+To debug application, follow these steps:
+
+1. Open **Command Palette** and enter **Run Web Simulator**.
+2. After the Web Simulator is launched, enter the path of the file in the box. For example, file:///C:/workspace/basic/index.html.
+
+    ![Debugging you application in the Web Simulator](media/tizen_web18.png)
+
+## Tizen Web with Code IntelliSense
+
+VS Code Extension for Tizen Web supports intelliSense, which includes the following features:
+
+- Code Completion
```

Comment:
From this line to line 295, about "Tizen Web with Code IntelliSense" section.

@sabeena21 I don't think your suggestion to change each features to title form and giving links to them is good.
The "Features" is 3rd depth form and to give 4th depth to each IntelliSenses and handling the links is too complicated. 
As the intelliSense are not that important article to explain in here, how about keeping it as it is now?
If you agree, please resolve your comments about giving links to "Tizen Web with Code IntelliSense" to reduce the comments.

Reply:
@mijin85cho, 
If using cross-reference would increase the load of the page. 
Can we get the content in the below format:

![image](https://user-images.githubusercontent.com/35525967/59998521-2f338880-964f-11e9-8f76-e89a3b88e8ec.png)

Reply:
@sabeena21 How about below format

## Tizen Web with Code IntelliSense
VS Code Extension for Tizen Web supports intelliSense. You can use these features to easily develop application using TAU.

### Features
 - Code completion
   Display a list of recommended values when specifying attributes or APIs.
   _image_
   
- Hover

...

Reply:
@sabeena21 also if you agree with this suggestion, how about to remove the comments about creating links to IntelliSense features?

Reply:
@mijin85cho, yes I agree with your suggestion.